server/fleet/hosts.go

```diff
@@ -93,6 +93,29 @@ func (s MacOSDiskEncryptionStatus) IsValid() bool {
 	}
 }
 
+// MDMBootstrapPackageStatus defines the possible statuses of the host's MDM bootstrap package,
+// which is derived from the status of the MDM command to install the bootstrap package.
+type MDMBootstrapPackageStatus string
+
+// TODO(Sarah): Should add a map of MDMBootstrapPackageStatus to the possible device responses i.e.
+// what gets recorded by nano) for the MDM command?
+// See https://developer.apple.com/documentation/devicemanagement/installenterpriseapplicationresponse
```

Comment:
I'm not sure I understand what you mean, is it to have something like:

```
MDMBootstrapPackageInstalled: Acknowledged
MDMBootstrapPackageFailed: Error
MDMBootstrapPackagePending: anything else
```
? If so I'm not sure if it would be worth it, given that we (also?) need this in SQL and it would be hard to reuse there?

Reply:
+1 I don't think it's worth it, at least for now.

Reply:
As far as reuse in SQL, I was thinking that it could be `map[MDMBootstrapPackageStatus][]string` and the SQL could then reuse it with `IN (?)`. I was thinking it could be useful for when we have changes like this [one](https://github.com/fleetdm/fleet/issues/11181). But if y'all think it is not worth the trouble, I'm ok with leaving it hard coded.